psiminer-cli/src/main/kotlin/config/Config.kt

```diff
@@ -15,7 +15,7 @@ data class Config(
     // ====== Pipeline configuration =====
     val filters: List<FilterConfig>,
     @SerialName("label") val labelExtractor: LabelExtractorConfig,
-    val storage: StorageConfig,
+    val storage: DatasetStorageConfig,
```

Comment:
And why do you rename it? Using `storage` both clear and short, I don't think that we need to explicitly point that this is storage for a dataset.